integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/smoke/YarnSmokeTest.java

```diff
@@ -64,21 +75,53 @@ public void testWhenCreatedYARNClusterShouldBeUpScaled(TestContext testContext)
                 .validate();
     }
 
-    @Test(dataProvider = TEST_CONTEXT_WITH_MOCK, enabled = false,
-            description = "A valid YARN cluster should be able to stopped and then started")
-    public void testWhenCreatedYARNClusterStoppedStarted(MockedTestContext testContext) {
-        testContext.given(StackTestDto.class)
-                .when(StackTestAction::create)
-                .await(STACK_AVAILABLE)
-                .when(StackTestAction::stop)
-                .await(STACK_STOPPED)
-                .when(StackTestAction::start)
+    @Test(dataProvider = TEST_CONTEXT, enabled = false, description = "Stop is not supported on YARN cloudplatform")
+    @Description(
+            given = "a valid YARN cluster",
+            when = "stopping then starting the cluster",
+            then = "the cluster should be available and successfully started"
+    )
+    public void testWhenCreatedYARNClusterStoppedStarted(TestContext testContext) {
```

Comment:
I know it's not up to you, but I really hope that Yarn will be reliable enough to introduce these tests to the build pipeline...

https://media3.giphy.com/media/uq6ILNBI6g3As/giphy.gif

Reply:
@lnardai or @sodre90 may can response this...may can convince you ;)